utils/composer-packages/src/Rector/AddReplacePackageRector.php

```diff
@@ -86,7 +89,7 @@ public function refactor(Node $node): ?Node
             foreach ($this->renamePackages as $renamePackage) {
                 $array->items[] = new ArrayItem(
                     new New_(
-                        new FullyQualified(RenamePackage::class),
+                        new FullyQualified('Rector\Composer\ValueObject\RenamePackage'),
```

Comment:
This might get accidentaly prefixed during rector/rector-src build and create invalid output, so the value should be `string` type.